server/events/event_parser.go

```diff
@@ -939,7 +939,12 @@ func (e *EventParser) ParseAzureDevopsRepo(adRepo *azuredevops.GitRepository) (m
 		host = "dev.azure.com"
 	}
 
-	cloneURL := fmt.Sprintf("https://%s/%s/%s/_git/%s", host, owner, project, repo)
+	cloneURL := ""
+	if strings.Contains(host, "visualstudio.com") {
```

Comment:
Could you comment that this is the modern non-legacy URL structure condition and the else structure is the older legacy URL structure?

Reply:
I've added a comment. Let me know if you want any changes.